Fixes #28 - Eliminate deprecation warnings

scotte opened the pull request.

evitalis commented:
@scotte It looks like this may be unmaintained at this point. If you want to issue this PR against my fork I will happily merge it to carry this project forward.

scotte commented:
Thanks @evitalis - I also have a fork where this is fixed, along with a couple of other bugs (keyboard focus when the search box is opened, fixed the footer to not encroach on the sidenav), but ultimately we ended up using the material theme since it provides sidenav for site contents as well as page contents and we discovered we really needed both.

My fork is https://github.com/scotte/cindercone if the other bugs fixed are of interest. Note I also switched the sidenav over to site contents, but this may or may not be what some folks want. Anyway, feel free to cherrypick any of my fixes from cidercone into your fork.
